Clappr/Classes/Base/Container.swift

```diff
@@ -52,8 +52,9 @@ open class Container: UIBaseObject {
         playbackOptions[kSourceUrl] = source
         playbackOptions[kMimeType] = mimeType
 
-        let playbackFactory = PlaybackFactory(loader: loader, options: playbackOptions)
+        self.playback?.removeFromSuperview()
```

Comment:
@brunopinheiro I think it would be better to call `self.playback?.destroy()` to destroy the old playback instance. Can you, please, check if it works?